index.html

```diff
@@ -41,11 +41,144 @@ <h2 class="headline-sub-title">I'm a Micronout</h2>
                         <li class="icon">
                             <a href="https://github.com/JuanDiegoGuerra" rel="noopener" target="_blank">
                                 <img src="img/social media icons.png" alt="github link">
-                            </a></li>
+                            </a>
+                        </li>
                     </ul>
                 </div>
                </footer>
             </section>
     </div>
+    <div id="my-recent-works">
+        <h1 class="recent-works-tittle">My Recent <br> Works</h1>
+        <div class="menu-line-recent-works"></div>
+        <div class="project-1">
+            <div class="first_project_img_div">
+            <img src="img/default_project_img.JPG" alt="first project image" class="first_project_image">
+            </div>
+            <div class="first-project">
+              <h3>
+                Hello-Microverse-Project <br>
+                First Week
+              </h3>
+                <div class="languages-grid">                    
+                        <div class="lang-list l1"><strong>Html5</strong></div>
+                        <div class="lang-list l2"><strong>Css3</strong></div>
+                        <div class="lang-list l3"><strong>Git</strong></div>
+                        <div class="lang-list l4"><strong>Github</strong></div>
+                </div>
+                <div class="external_link">
+                    <a href="#">See Project</a>
+                </div>
+            </div>
+        </div>
+        </div>
+        <div class="project-2">
+            <div class="first_project_img_div">
+            <img src="img/default_project_img.JPG" alt="first project image" class="first_project_image">
+            </div>
+            <div class="first-project">
+              <h3>
+                Setup-Portfolio-Mobile-1st <br>
+                First Week
+              </h3>
+                <div class="languages-grid">                    
+                        <div class="lang-list l1"><strong>Html5</strong></div>
+                        <div class="lang-list l2"><strong>Css3</strong></div>
+                        <div class="lang-list l3"><strong>Git</strong></div>
+                        <div class="lang-list l4"><strong>Github</strong></div>
+                </div>
+                <div class="external_link">
+                    <a href="#">See Project</a>
+                </div>
+            </div>
+        </div>       
+    </div>
+    <div id="about-me">
+        <div class="about-me-bg">
+        <h1 class="about-me-tittle">About me</h1>
```

Comment:
- [ ] Using only one `H1` creates a hierarchy and gives your page a more logical structure. It makes it easy for readers (and those using screen readers) to skim and understand what your page is about. Kindly use a `hx` instead 👍